packages/backend/vitest.config.integration.ts

```diff
@@ -0,0 +1,27 @@
+import * as path from 'node:path';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+    test: {
+        name: 'integration-tests',
+        include: ['src/ee/**/*integration.test.ts'],
+        exclude: ['node_modules', 'dist'],
+        environment: 'node',
+        testTimeout: 60000,
+        hookTimeout: 60000,
+        teardownTimeout: 60000,
+        globals: true,
+        setupFiles: ['./src/vitest.setup.integration.ts'],
+        env: {
+            TZ: 'UTC',
```

Comment:
I noticed it being set in package.json too

Reply:
done